src/textual/__init__.py

```diff
@@ -26,19 +27,7 @@
 
 LogCallable: TypeAlias = "Callable"
 
-
-def __getattr__(name: str) -> str:
-    """Lazily get the version from whatever API is available."""
-    if name == "__version__":
-        try:
-            from importlib.metadata import version
-        except ImportError:
-            import pkg_resources
-
-            return pkg_resources.get_distribution("textual").version
-        else:
-            return version("textual")
-    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+__version__ = version("textual")
```

Comment:
Even if we use `importlib.metadata`'s `version`, we probably still want it to be lazily computed to make sure Textual's startup time isn't impacted by the retrieval of the version when it's not needed.
Does that make sense?

Reply:
If I recall that was the reason for doing it lazily. It was surprisingly slow.

Docs say that importlib.metadata was adding in 3.8. We still support 3.7, so I don't think we could accept this PR.

Reply:
> We still support 3.7

I thought this was dropped following #3766?

Reply:
You're right. Sorry, I'm a version behind.

Reply:
In that case, no objections if we restore the lazy behaviour!

Reply:
restored the lazy import!